### PR TITLE
Added Code

### PR DESCRIPTION
**Pull request by Anubhav-Ghosh1**

### CODE REVIEW

File: LeetCode Problems/301 to 400/331. Verify Preorder Serialization of a Binary Tree/331. Verify Preorder Serialization of a Binary Tree.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+#include <bits/stdc++.h>
+using namespace std;
+
```
Comment (md-shamim-ahmad) [inline LeetCode Problems/301 to 400/331. Verify Preorder Serialization of a Binary Tree/331. Verify Preorder Serialization of a Binary Tree.cpp]: remove those line from here
```cpp
#include <iostream>
#include <bits/stdc++.h>
using namespace std;

```

**Review by md-shamim-ahmad — CHANGES_REQUESTED** [review]
Though the file name ```ReadMe.md``` and ```README.md``` have no effect.
But I do not have any plane to add a ```.md``` file name which is not same as ```README.md```
So rename the file ```Readme.md``` to ```README.md``` please

**Comment by Anubhav-Ghosh1** [comment]
Completed changes

**Comment by Anubhav-Ghosh1** [comment]
Check it please

**Comment by Anubhav-Ghosh1** [comment]
I have completed the changes

**Comment by md-shamim-ahmad** [comment]
I think you code have bug
```
9,3,4,#,#,1,#,#,2,#,6,#,#
```
Your given solution code **output:** ``` false``` but the answer will be ```true```
Please recheck It.

**Comment by Anubhav-Ghosh1** [comment]
Did the changes Accepted
Runtime: 0 ms
Case 1
Case 2
Case 3
Input
a1

preorder =
"9,3,4,#,#,1,#,#,2,#,6,#,#"
Stdout
0
Output
true
Expected
true

**Review by md-shamim-ahmad — CHANGES_REQUESTED** [review]
Although the code passed the test case.
But the code does not follow the ```clear code format```.
Use ```clear and meaningful``` **variable** names.

**Comment by Anubhav-Ghosh1** [comment]
Made changes

File: LeetCode Problems/301 to 400/331. Verify Preorder Serialization of a Binary Tree/331. Verify Preorder Serialization of a Binary Tree.cpp [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        if(str[0]=='#' and str.size()!=1)
+        return 0;
+
+        for(int i=0;i<str.size();i+=1){
+            count-=1;
+            if(count<0)
+            return 0;
+
+            if(str[i]!='#'){
+                c+=2;
```
Comment (md-shamim-ahmad) [inline LeetCode Problems/301 to 400/331. Verify Preorder Serialization of a Binary Tree/331. Verify Preorder Serialization of a Binary Tree.cpp]: where you declare the variable ```c```

**Comment by md-shamim-ahmad** [comment]
You can use this code 
```cpp
class Solution {
public:
    bool isValidSerialization(string str) {
        int count = 1;
        if (str.size() == 0)
            return 1;
        if (str[0] == '#' and str.size() != 1)
            return 0;
        for (int i = 0; i < str.size(); i++) {
            count -= 1;
            if (count < 0)
                return 0;
            if (str[i] != '#') {
                count += 2;
            }
            i++;
            while (i < str.size() and str[i] != ',') {
                i++;
            }
        }
        if (count == 0)
            return 1;
        return 0;
    }
};
```

**Comment by Anubhav-Ghosh1** [comment]
Done

**Review by md-shamim-ahmad — APPROVED** [review]
Thanks for your contribution :)

**Comment by Anubhav-Ghosh1** [comment]
I have made an new code

**Pull request merged**